be/src/agent/user_resource_listener.cpp

```diff
@@ -64,16 +61,15 @@ void UserResourceListener::update_users_resource(int64_t new_version) {
     }
     // Call fe to get latest user resource
     Status master_status;
-    // using 500ms as default timeout value    
+    // using 500ms as default timeout value
     FrontendServiceConnection client(_exec_env->frontend_client_cache(),
-                                   _master_info.network_address,
-                                   config::thrift_rpc_timeout_ms, 
-                                   &master_status);
+                                     _master_info.network_address, config::thrift_rpc_timeout_ms,
+                                     &master_status);
     TFetchResourceResult new_fetched_resource;
-    if (!master_status.ok()) { 
-        LOG(ERROR) << "Get frontend client failed, with address:" 
-            << _master_info.network_address.hostname << ":" 
-            << _master_info.network_address.port;
+    if (!master_status.ok()) {
+        LOG(ERROR) << "Get frontend client failed, with address:"
+                   << _master_info.network_address.hostname << ":"
```

Comment:
Similar to the previous indentation problem, alignment with the "<<" on the previous line should not be mandatory, 8 spaces should be allowed.

BTW: here there is only 4 spaces, so need to be changed one of the two ways.

Reply:
Unfortunately, I can't find a way to make clang-format accept both the "align <<" and "ident with 8 spaces" style. I think the "align <<" style is preferable, is that OK?